src/config/config.ts

```diff
@@ -0,0 +1,14 @@
+import { DEV, STAGE, PROD } from "./constants";
+import { configConstants } from "../amo/types";
+
+const environment = process.env.NODE_ENV;
```

Comment:
can you add a section to the readme explaining how to switch environments

Reply:
Is there a way to switch the environment via the command line or by setting ENV variables in the shell?

Reply:
Can you log a follow-up issue for it?  Also checkout how addons-frontend does it (but no need to exactly copy the method there)